integreatly-mobile-developer-console/0.2.6/mobile-developer-console-operator.v0.2.6.clusterserviceversion.yaml

```diff
@@ -51,7 +51,7 @@ spec:
                 - name: IDM_DOCUMENTATION_URL
                 - name: SYNC_DOCUMENTATION_URL
                 - name: MSS_DOCUMENTATION_URL
-                image: quay.io/aerogear/mobile-developer-console-operator:0.2.6
+                image: quay.io/aerogear/mobile-developer-console-operator:ae712e3
```

Comment:
why a hash instead of a tag here, is that `0.2.6` a floating tag?

Reply:
No, the 0.2.6 is the latest release tag for the MDC operator but it doesn't include the changes for the mobile client admin role as it was done to unblock the install of the MDC operator, the changes are currently only available in master and to avoid pointing the image to master tag I asked @grdryn to create a label for it